backup/fileSaveLoad.js

```diff
@@ -71,27 +71,25 @@ document.getElementById('file-all-styles').addEventListener('click', function ()
   });
 });
 
-document.getElementById('unfile-all-styles').addEventListener('click', function () {
-  loadFromFile(STYLISH_DUMPFILE_EXTENSION).then(function (rawText) {
-    var json = JSON.parse(rawText);
-    var i = 0, nextStyle;
+document.getElementById('unfile-all-styles').addEventListener('click', () => {
+  loadFromFile(STYLISH_DUMPFILE_EXTENSION).then(rawText => {
+    const json = JSON.parse(rawText);
+    const numStyles = json.length;
 
-    function done() {
-      window.alert(i + ' styles installed/updated');
-      location.reload();
-    }
+    invalidateCache(true);

```

Comment:
Please add this function to globals; line 1
/* globals getStyles, saveStyle, invalidateCache */